README.md

```diff
@@ -120,8 +120,9 @@ npm install autotrack
 // In your JavaScript code
 import 'autotrack';
 ```
+_Note autotrack's source is published as ES2015, and you will need to make sure you're not excluding it from compilation. See #137 for more details._
```

Comment:
Can you format this to be the same as other notes in the README? Where the word "note" is in bold. Also please link to the issue.